### PR TITLE
Fix some hover issues

### LINKED ISSUE #1373: Add hover state to rows on search results

### LINKED ISSUE #1382: Missing hoverovers in the relations panel of the inspector

### PR DESCRIPTION
**Pull request by anthonydillon**
Fixes #1382 
Fixes #1373

### CODE REVIEW

**Comment by anthonydillon** [comment]
## QA
Check the following issues are fixed:
- https://github.com/juju/juju-gui/issues/1382
- https://github.com/juju/juju-gui/issues/1373

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4528/
Test PASSed.

File: jujugui/static/gui/src/app/components/inspector-relations-item/_inspector-relations-item.scss [hunk 1]
```diff
@@ -12,6 +12,10 @@
         border-bottom: 1px solid $mid-grey;
         line-height: 1.5;
         cursor: pointer;
+
+        &:hover {
+            background-color: #f7f7f7;
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/inspector-relations-item/_inspector-relations-item.scss]: We have a variable for this `$hover-background`.

File: jujugui/static/gui/src/app/components/search-results/_search-results.scss [hunk 2]
```diff
@@ -60,6 +60,10 @@
                     @media only screen and (min-width : 768px) {
                         padding-bottom: 8px;
                     }
+
+                    &:hover {
+                        background-color: #f7f7f7;
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/search-results/_search-results.scss]: You can use `$hover-background`.

File: jujugui/static/gui/src/app/components/inspector-relations-item/_inspector-relations-item.scss [hunk 3]
```diff
@@ -12,6 +12,10 @@
         border-bottom: 1px solid $mid-grey;
         line-height: 1.5;
         cursor: pointer;
+
+        &:hover {
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/inspector-relations-item/_inspector-relations-item.scss]: Can you please add `&:focus, &:active`?

File: jujugui/static/gui/src/app/components/search-results/_search-results.scss [hunk 4]
```diff
@@ -60,6 +60,10 @@
                     @media only screen and (min-width : 768px) {
                         padding-bottom: 8px;
                     }
+
+                    &:hover {
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/components/search-results/_search-results.scss]: Can you please add `&:focus, &:active`?

**Comment by huwshimi** [comment]
@anthonydillon one QA issue: I get the hover for recommended search results, but not for the community ones.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4538/
Test PASSed.

**Comment by hatched** [comment]
:+1: :ok:
Thanks!
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**